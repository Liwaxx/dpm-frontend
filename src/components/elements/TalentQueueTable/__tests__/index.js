import React from 'react';
import renderer from 'react-test-renderer';
import Component from '../index';

jest.mock('../../../elements/Text');

describe('TalentScoreCard', () => {
  it('renders correctly', () => {
    const data = [
      { name: 'tes', version: '1' },
    ];
    const tree = renderer
      .create(<Component data={data} />)
      .toJSON();

    expect(tree).toMatchSnapshot();
  });

  it('renders with custom class', () => {
    const tree = renderer
      .create(<Component className="test" />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});