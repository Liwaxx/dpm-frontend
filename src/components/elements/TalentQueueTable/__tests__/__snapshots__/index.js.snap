// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TalentScoreCard renders correctly 1`] = `
<table
  className="Component-root-3"
>
  <thead>
    <tr>
      <th />
      <th />
    </tr>
  </thead>
  <tbody>
    <tr>
      <td />
      <td />
    </tr>
  </tbody>
</table>
`;

exports[`TalentScoreCard renders with custom class 1`] = `
<table
  className="Component-root-3 test"
>
  <thead>
    <tr>
      <th />
      <th />
    </tr>
  </thead>
  <tbody />
</table>
`;
